src/libsync/vfs/cfapi/shellext/CMakeLists.txt

```diff
@@ -172,6 +174,7 @@ add_library(CfApiShellExtensions MODULE
     ${CMAKE_SOURCE_DIR}/src/common/shellextensionutils.cpp
     customstateprovider.cpp
     CfApiShellIntegration.def
+    CfApiShellIntegration.rc
```

Comment:
```suggestion
    CfApiShellExtensions.rc
```

Reply:
it is the opposite `.def` and `.rc` files should have the same base name